.prettierignore

```diff
@@ -15,7 +15,8 @@ lerna.json
 .github/actions/issue-labeler/lib/index.js
 packages/next-swc/crates/next-core/js/src/compiled/**/*
 packages/next-swc/crates/**/tests/**/output*
-packages/next-swc/target/**/*
+packages/next-swc/crates/core/tests/loader/issue-32553/input.js
```

Comment:
I put `prettier-ignore` comments on every single line of this file and prettier still broke the lines, so I had to add it here